lib/tasks/publishing_api.rake

```diff
@@ -32,6 +32,56 @@ namespace :publishing_api do
       ))
     end
   end
+
+  desc "Send publishable item links to Publishing API"
+  task publishing_api_patch_links: :environment do
+    $stdout.puts "# Sending all items' links to Publishing API"
+    $stdout.puts "# Ruby code is output for failed operations"
+    $stdout.puts
+
+    def patch_links(item)
+      content_id = item.content_id
+      links = PublishingApiPresenters.presenter_for(item).links
+      if links
+        Whitehall.publishing_api_v2_client.patch_links(content_id, {links: links})
+      else
+        $stdout.puts "# No links for item with content_id: #{content_id}"
+      end
+    rescue => err
```

Comment:
It's probably best to only rescue `GdsApi` exceptions here, so we know its something external.